swap_meet/vendor.py

```diff
@@ -1,2 +1,80 @@
 class Vendor:
-    pass
+    
+    def __init__(self, inventory = None):
+        """ 
+        Initializes the inventory attribute
+        """
+        self.inventory = [] if not inventory else inventory # Turnery conditional expression
+
+    def add(self, item):
+        """
+        Adds item to inventory
+        """
+        self.inventory.append(item)
+        return item
+        
+    def remove(self, item):
+        """
+        Removes item from inventory
+        """
+        item_exists = False
+        if item in self.inventory:
+            self.inventory.remove(item)
+            item_exists = True
+        return item if item_exists else item_exists
```

Comment:
I think this could be a bit simpler

```suggestion
        if item not in self.inventory:
            return False
        self.inventory.remove(item)
        return True
```